### ISSUE TITLE
Cannot convert to kanji on the comment field of Facebook live

### ISSUE THREAD

**Issue by justoneplanet** [google/mozc#386]
### Devices
- Nexus 5x
### What steps will reproduce the problem?
1. Launch facebook official app.
2. Touch the "Live".
3. Touch the balloon icon to comment.
4. Type text.
### What is the expected output?

Composing hiragana text will be inputted to convert to kanji.
### What do you see instead?

Committed text is inputted. so we cannot convert to kanji.
### What version of the product are you using? On what operating system?

Android 7.0
- mozc 2.16.2038.103
- Google Japanese Input 2.18.2581.3.129228829-arm64-v8a

**Comment by matsuza** [comment]
Thanks for reporting.
Which product did you use when you reproduced the issue? Both?

**Comment by justoneplanet** [comment]
Sorry for my late reply.

> Which product did you use when you reproduced the issue? Both?

Both

This problem will be reproduced on the text field of facebook app.

https://play.google.com/store/apps/details?id=com.facebook.katana

After I reported this issue, I had kept to test using another IME.
And most Japanese IME like ATOK failed to convert to kanji on the facebook live text field.
Thus, I think this problem will be caused by facebook app implementation now.
